cssom-view-1/Overview.bs

```diff
@@ -460,8 +463,14 @@ When a user agent is to <dfn for="viewport" export>perform a scroll</dfn> of a <
 1. Let <var>element</var> be <var>doc</var>'s root element if there is one, null otherwise.
 1. <a for="/">Perform a scroll</a> of the <a>viewport</a>'s <a>scrolling box</a> to its current scroll position + (<var>layout dx</var>, <var>layout dy</var>) with <var>element</var> as the
     associated element, and <var>behavior</var> as the scroll behavior.
+    Let <var>scrollPromise1</var> be the promise returned from this step.
 1. <a for="/">Perform a scroll</a> of <var>vv</var>'s <a>scrolling box</a> to its current scroll position + (<var>visual dx</var>, <var>visual dy</var>) with <var>element</var> as the associated
     element, and <var>behavior</var> as the scroll behavior.
+    Let <var>scrollPromise2</var> be the promise returned from this step.
+1. Wait until either both <var>scrollPromise1</var> and <var>scrollPromise2</var> get resolved with true, or any of them settles in any other state (i.e. gets resolved with false or gets rejected).
```

Comment:
I think we return scrollPromise2 immediately, but wait on this before resolving the promise.